modules/core/shared/src/main/scala/weaver/internals/ExpectSame.scala

```diff
@@ -43,7 +29,7 @@ private[weaver] trait ExpectSame {
   def same[A](
       expected: A,
       found: A)(
-      implicit eqA: Eq[A] = Eq.fromUniversalEquals[A],
-      showA: Show[A] = Show.fromToString[A],
-      loc: SourceLocation): Expectations = eql(expected, found)
+      implicit loc: SourceLocation): Expectations = eql(expected, found)(
```

Comment:
Mmm what if we still passed the comparison as an implicit parameter with the default ? Now that I think a bit more about it, the default instance was the main motivation for having a distinction in the previous version between `same` and `eql`, whilst still allowing for injectable behaviour. Retaining this property would be valuable I think 

```scala 
implicit comparisonA: Comparison[A] = Comparison.fromEq(Eq.fromUniversalEquals, Show.fromToString)
```